build.sbt

```diff
@@ -44,10 +44,15 @@ lazy val microservice = Project(appName, file("."))
   )
   .settings(resolvers += Resolver.jcenterRepo)
   .settings(scalastyleSettings)
+  .settings(
+    scalafmtDetailedError := true,
+    scalafmtPrintDiff := true,
+    scalafmtFailOnErrors := true
+  )
 
 lazy val it = project
   .enablePlugins(PlayScala)
   .dependsOn(microservice % "test->test")
   .settings(libraryDependencies ++= Seq("uk.gov.hmrc" %% "bootstrap-test-play-30" % bootstrap % Test))
 
-addCommandAlias("runAllChecks", ";clean;compile;coverage;test;it/test;scalastyle;Test/scalastyle;coverageReport")
+addCommandAlias("runAllChecks", ";clean;compile;coverage;test;scalafmtCheckAll;it/test;scalastyle;Test/scalastyle;coverageReport")
```

Comment:
Better to move scalafmtCheckAll command just before scalastyle and after it/test